src/generator/getAllRules.ts

```diff
@@ -0,0 +1,56 @@
+import { RulesValues } from '../rules/rules.module';
+
+import * as fs from 'fs';
+export function getAllRules(): object {
+  const path = require('path');
+
+  let result: any = {};
+
+  const rulesPath = RulesValues.map(r => {
+    return (
+      r.name[0].toLowerCase() + r.name.substr(1, r.name.length - 5) + '.rule.ts'
```

Comment:
Instead of concatenation, you can maybe use string templates
```suggestion
      `${r.name[0].toLowerCase()}${r.name.substr(1, r.name.length - 5)}.rule.ts`
```